sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -1832,7 +1832,7 @@
                         "value": ""
                       }
                     ],
-                    "content": "$#######",
+                    "content": "{{data.disbursement?.disbursement1Date ? data.disbursement.disbursement1Date : ''}}",
```

Comment:
The expression could be simplified as below.
```
{{data.disbursement?.disbursement1Date ?? : ''}}
````

Reply:
I believe we can also use `{{data.disbursement?.disbursement1Date}}  ??  ' '`, both ways will work, not sure about the better approach.

Reply:
avoiding ?? operator for the benefit that it is not supported in 2 browsers.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
@dheepak-aot, this code is from the API, it will never gonna reach or be executed by the browser, only for Nodejs. The argument does not apply here.